java/src/jmri/managers/AbstractTurnoutManager.java

```diff
@@ -283,6 +283,8 @@ public String getNextValidAddress(@Nonnull String curAddress, @Nonnull String pr
                     return Integer.toString(iName);
                 }
             }
+            // feedback when next address is also in use
+            log.warn("10 hardware addresses starting at {} already in use. No new Turnouts added", curAddress);
```

Comment:
Add a warning in console to explain why no new items were added. Also for Sensors (Lights don't look ahead and already provide status bar in pane)